tests/MyGame/Example/Vec3.cs

```diff
@@ -59,15 +59,17 @@ public struct Vec3 : IFlatbufferObject
   }
   public static Offset<MyGame.Example.Vec3> Pack(FlatBufferBuilder builder, Vec3T _o) {
     if (_o == null) return default(Offset<MyGame.Example.Vec3>);
+    var _test3_a = _o.Test3.A;
```

Comment:
why is this variable introduced? Please stick `_o.Test3.A` directly in the call below.

Reply:
I think this simplified the codeine since in some cases this variable needs to be reused. cc: @mugisoba

Reply:
Since structures can contain fixed-size arrays, they are once assigned here as variables for code commonality. 
If fixed-size array is found, additional initialization code is generated.